stories/core_components/Kpi.tsx

```diff
@@ -45,4 +46,15 @@ storiesOf('Core components/KPI', module)
                 projectId={'storybook'}
             />
         )
+    ))
+    .add('with Germany number format', () => (
+        screenshotWrap(
+            <Kpi
+                measure={'/gdc/md/storybook/obj/1'}
+                projectId={'storybook'}
+                separators={GERMANY_NUMBER_FORMAT}
```

Comment:
I think you mean 'Kpi'.
I see that Kpi has `format` property for number, which is at same level with `separators`. So that's why I don't wrap `separators` under `config`.

Another solution is that `config` will wrap `format` and `separators`. But I think it's not good way.